docs/source/user-guide/sql/select.md

```diff
@@ -261,6 +262,14 @@ Example:
 SELECT a, b, MAX(c) FROM table GROUP BY a, b HAVING MAX(c) > 10
 ```
 
+## QUALIFY clause
+
```

Comment:
It would be nice if we could provide a little context here (like explaining that the QUALIFY statement can refer to the output of window functions)

However, I think this is consistent with the sparse docs for HAVING 

Maybe we can make a follow on PR to improve the documentation with a sentence or two for each clause